templates/projects/list.tmpl

```diff
@@ -38,30 +38,41 @@
 		<div class="milestone list">
 			{{range .Projects}}
 				<li class="item">
-					{{svg .IconName}} <a href="{{.Link}}">{{.Title}}</a>
-					<div class="meta">
-						{{$closedDate:= TimeSinceUnix .ClosedDateUnix $.locale}}
-						{{if .IsClosed}}
-							{{svg "octicon-clock"}} {{$.locale.Tr "repo.milestones.closed" $closedDate | Safe}}
-						{{end}}
-						<span class="issue-stats">
-							{{svg "octicon-issue-opened" 16 "gt-mr-3"}}
-							{{$.locale.PrettyNumber .NumOpenIssues}}&nbsp;{{$.locale.Tr "repo.issues.open_title"}}
-							{{svg "octicon-check" 16 "gt-mr-3"}}
-							{{$.locale.PrettyNumber .NumClosedIssues}}&nbsp;{{$.locale.Tr "repo.issues.closed_title"}}
-						</span>
+					<div class="header">
+						<h3 class="gt-df gt-ac gt-m-0 gt-fw">
```

Comment:
Is it better to use `<h3 header>` instead of `<div header><h3>`? I don't find the "div" is useful.